src/index.js

```diff
@@ -520,15 +519,23 @@ function createConfig(options, entry, format, writeMeta) {
 							],
 						],
 					}),
-					// Only used for async await
 					babel({
```

Comment:
There's a redundant babel call above this.

Reply:
not really redundant, the babel call is used to transpile node_modules to transform definitions like NODE_ENV.